gammapy/scripts/tests/test_image_pipe.py

```diff
@@ -64,9 +64,8 @@ def test_image_pipe(tmpdir):
     mosaic = MosaicImage(image, energy_band=energy_band, offset_band=offset_band, data_store=data_store,
                          obs_table=data_store.obs_table, exclusion_mask=exclusion_mask)
     mosaic.make_images(make_background_image=True, for_integral_flux=True, radius=10.)
-    # TODO: this fails on some machines, for unknown reasons.
-    #assert_equal(mosaic.maps['counts'].data.sum(), 2334.0)
-    #assert_almost_equal(mosaic.maps['bkg'].data.sum(), 1987.2088283446637)
-    #assert_almost_equal(mosaic.maps['exposure'].data.sum(), 5.9292301086257267e17)
-    #assert_almost_equal(mosaic.maps['significance'].lookup(center), 33.699068269123039)
-    #assert_almost_equal(mosaic.maps['excess'].data.sum(), 346.79117165533626)
+    assert_allclose(mosaic.maps['counts'].data.sum(), 2334.0, atol=3)
```

Comment:
@JouvinLea - It's OK as is. Nothing to do for you.
Test tolerances could be set more strict, but that's a small improvement that can be done any time in the future.

@adonath - No, that's not how it works. For this bin edge effect, the event reco and PSF doesn't matter. Most of our events are diffuse proton background events.
Rounding event positions isn't a good idea ... what would you round to? Nearest decimal or binary to some digit?

Filling in bins is just an operation where arbitrary small changes in position can result in a bin moving from one bin to the next or outside an image. Science results have to be produced so that they don't depend on the binning. (They do a little with hard kernels like Li&Ma, that's a major reason why TS maps from smooth kernels are sometimes useful.)